adafruit_button.py

```diff
@@ -118,68 +180,14 @@ def _create_body(self):
             if self.shadow:
                 self.append(self.shadow)
 
-    RECT = const(0)
-    ROUNDRECT = const(1)
-    SHADOWRECT = const(2)
-    SHADOWROUNDRECT = const(3)
-
-    def __init__(
-        self,
-        *,
-        x,
-        y,
-        width,
-        height,
-        name=None,
-        style=RECT,
-        fill_color=0xFFFFFF,
-        outline_color=0x0,
-        label=None,
-        label_font=None,
-        label_color=0x0,
-        selected_fill=None,
-        selected_outline=None,
-        selected_label=None
-    ):
-        super().__init__(x=x, y=y)
-        self.x = x
-        self.y = y
-        self._width = width
-        self._height = height
-        self._font = label_font
-        self._selected = False
-        self.name = name
-        self._label = label
-        self.body = self.fill = self.shadow = None
-        self.style = style
-
-        self._fill_color = _check_color(fill_color)
-        self._outline_color = _check_color(outline_color)
-        self._label_color = label_color
-        self._label_font = label_font
-        # Selecting inverts the button colors!
-        self._selected_fill = _check_color(selected_fill)
-        self._selected_outline = _check_color(selected_outline)
-        self._selected_label = _check_color(selected_label)
-
-        if self.selected_fill is None and fill_color is not None:
-            self.selected_fill = (~self._fill_color) & 0xFFFFFF
-        if self.selected_outline is None and outline_color is not None:
-            self.selected_outline = (~self._outline_color) & 0xFFFFFF
-
-        self._create_body()
-        if self.body:
-            self.append(self.body)
-
-        self.label = label
-
     @property
-    def label(self):
+    def label(self) -> Optional[Union[Any, str]]:
```

Comment:
I'm not sure we need `Any` here for the return type. Are there cases where the return will be something other than a `string` or `None`?